### ISSUE TITLE
 Couldn't open the file or directory "H:\wlsdev\fedora1\rootfs\usr\share\terminfo\A\"

### ISSUE THREAD

**Issue by ozzioma** [DDoSolitary/LxRunOffline#31]
>  Couldn't open the file or directory "H:\wlsdev\fedora1\rootfs\usr\share\terminfo\A\"

I am having this error while installing using the following versions

Windows 10 Fall Creators Update x64, 1709
LxRunOffline 2.2.2
fedora-rawhide-x86_64-20180303.tar.xz

LxRunOffline is installed to c:\tools

This is my install command
lxrunoffline install -n fedora1 -d h:\wlsdev\fedora1 -f h:\wlsdev\fedora-rawhide-x86_64-20180303.tar.xz -r .

I have tried setting the -d option to drive C and other partitioned drives, but the error persists.
What could be wrong?

**Comment by DDoSolitary** [comment]
Have you restarted your computer after installing running `choco install lxrunoffline`?

**Comment by AjawadMahmoud** [comment]
I can confirm I had the same issue and a restart resolved it.
Can you explain the reason?

**Comment by DDoSolitary** [comment]
@AjawadMahmoud The installation script changes a registry value, which require a restart to take effect. (See #21.)

**Comment by MrChris79** [comment]
I am getting a similar if not same error when running the below command. I am not using choco however. This appears to be the case only when downloading/installing stretch from url in wiki. I do not get this error when downloading/installing sid from urls in wiki. Have tried reboots. happens on at least 2 pc's iv tried it on so far. Could my download be corrupted or something? the SHA1 hash for `rootfs.tar.xz` I download is `B86DE04C0E24D16927EAC089A7946B9087279AF5`. I tried to extract `pam.7.gz` from `rootfs.tar.gz` using WinRar and it complained about not being able to create a symbolic link to `pam.7.gz`.

`LxRunOffline.exe v2.2.2`
`LxRunOffline.exe install -n debian-stretch -f C:\linux\debian\stretch\debian_stretch_rootfs.tar.xz -d C:\linux\debian\stretch\root'`

`[ERROR] Couldn't open the file or directory "C:\linux\debian\stretch\root\rootfs\usr\share\man\man7\pam.7.gz".`

~MC

**Comment by ozzioma** [comment]
It works after a restart.
Thanks.

**Comment by DDoSolitary** [comment]
@MrChris79 That's really strange. I just checked the root.tar.gz file of sid and found out that it contains the same files as stretch in `/usr/share/man/man7/`.

```
-rw-r--r-- 1 root root 2142 May 27  2017 PAM.7.gz
-rw-r--r-- 1 root root  508 May 16  2017 bash-builtins.7.gz
lrwxrwxrwx 1 root root   31 Mar 12 08:00 builtins.7.gz -> /etc/alternatives/builtins.7.gz
lrwxrwxrwx 1 root root    8 May 27  2017 pam.7.gz -> PAM.7.gz
-rw-r--r-- 1 root root 9934 Nov 24 17:22 tc-hfsc.7.gz
-rw-r--r-- 1 root root 4111 Dec 28 17:47 term.7.gz
```

I think your downloaded file isn't corrupted because I downloaded the root.tar.gz file and get exactly the same SHA1 digest. The only difference is that I can install it successfully with `LxRunOffline install` 😳 

I'll keep investigating what's actually happening.

**Comment by MrChris79** [comment]
So at is turns out I had to basically reset the reg/gpo settings. on then back off or off then back on etc.. with a few reboots, but its working now. I wonder if it could have maybe been a windows update that is causing this or something. The 2 PCs iv tried it on so far are not using Insider Preview versions of Windows 10. Glad I figured it out and thanks for looking into it. Looking forward to the future of WSL/LxRunOffline. 

I am working on a project to make a "semi-portable" version of basically lxrunoffline/wsl without reinventing the wheel. I have made an alpha version that is semi broke at the moment but does kinda work (just not the way I want). The Idea I am working from is to make WSL portable in the sense that it can be "disabled/enabled" between OS Reloads without reinstalling all the *nix distro apps such as apache etc. Not portable in the sense of running from usb on many computers but rather keeping all settings etc.. isolated in one main folder structure. However usb portability may be something I work on if the efforts are worth it :)

Thanks,

~MC
